changelogs/fragments/2120-zos_lineinfile-Added-return-content.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - zos_lineinfile - Added return_content in logs for the zos_lineinfile module.
```

Comment:
Following the convention we use to write changelogs (https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments) this should be written in the following manner:
```
bugfixes:
  - ansible_play_batch - variable included unreachable hosts. Fix now saves unreachable hosts between plays by adding them to the PlayIterator's ``_play._removed_hosts`` (https://github.com/ansible/ansible/issues/66945).
```

So I think this should be rewritten to:
```
Return values ``return_content`` and ``backup_name`` were not always being returned. Fix now ensure that these values are always present in the module's response.
```